extension/src/connect/token.ts

```diff
@@ -0,0 +1,8 @@
+export const STUDIO_ACCESS_TOKEN_KEY = 'dvc.studioAccessToken'
+
+export const isStudioAccessToken = (text?: string): boolean => {
+  if (!text) {
+    return false
+  }
+  return text.startsWith('isat_') && text.length === 54
```

Comment:
[F] This was the basic format that I gathered from each token. With the rebrand, this may change but lets see.

Reply:
My token was actually `53` characters long 😅 

https://user-images.githubusercontent.com/43496356/219052994-8eb69fc6-fd64-48b0-af67-25b3292b1b4b.mov

Reply:
I'll relax the requirement